Vagrantfile

```diff
@@ -8,15 +8,17 @@ require "#{VAGRANTFILE_DIR}/vagrant/lib/forklift"
 
 def migrate_boxes!
   old = "#{VAGRANTFILE_DIR}/boxes.d/99-local.yaml"
+  older = "#{VAGRANTFILE_DIR}/boxes.yaml"
   new = "#{VAGRANTFILE_DIR}/vagrant/boxes.d/99-local.yaml"
 
-  return if File.symlink?(old) || !File.exist?(old)
+  return if File.symlink?(old) || !File.exist?(old) || !File.exist?(older)
```

Comment:
I'd make it `return if (File.symlink?(old) || !File.exist?(old)) && (File.symlink?(older) || !File.exist?(older))`